src/components/M3FabMenu/m3FabMenu.module.scss

```
@@ -0,0 +1,153 @@
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+.stack {
+    // Absolute, not a flex sibling: an in-flow stack pushes everything above the
+    // FAB — the search row was shoved off-screen every time the menu opened.
+    // The menu lies ON TOP of the rows instead, so nothing but the FAB's own
+    // icon changes when you open it.
+    position: absolute;
+    // 6px clear of the FAB, matching the bar row's own inset.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)